doc/en/example/simple.rst

```diff
@@ -761,6 +761,36 @@ and run it::
 You'll see that the fixture finalizers could use the precise reporting
 information.
 
+PYTEST_CURRENT_TEST environment variable
+----------------------------------------
+
+.. versionadded:: 3.2
+
+Sometimes a test session might get stuck and there might be no easy way to figure out
+which test got stuck, for example if pytest was run in quiet mode (``-q``) or you don't have access to the console
+output. This is particularly a problem if the problem helps only sporadically, the famous "flaky" kind of tests.
+
+``pytest`` sets a ``PYTEST_CURRENT_TEST`` environment variable when running tests, which can be inspected
+by process monitoring utilities or libraries like `psutil <https://pypi.python.org/pypi/psutil>`_ to discover which
+test got stuck if necessary.
```

Comment:
I assume that'd be something like `psutil.Process(pid).environ()['PYTEST_CURRENT_TEST']` then? Maybe add that as an example?